src/analysis/retail/evoker/augmentation/modules/features/BuffTargetHelper/BuffTargetHelper.tsx

```diff
@@ -88,6 +94,14 @@ class BuffTargetHelper extends Analyzer {
   mrtTwoTargetPrescienceHelperNote: string = '';
   mrtFourTargetPrescienceHelperNote: string = '';
 
+  filterBossDamage: boolean = false;
+  nameFilter: string = '';
+  bossFilter: string = this.owner.report.enemies
+    .filter((enemy) => enemy.type === 'Boss')
+    .map((enemy) => `${enemy.guid}`)
+    .join(',');
+  abilityFilter: string = ABILITY_BLACKLIST.map((id) => `${id}`).join(', ');
```

Comment:
can  just be `ABILITY_BLACKLIST.join(', ')` since that converts it to a string anyways